### PR TITLE
[TTAHUB-1348] Topic guidance

### PR DESCRIPTION
**Pull request by AdamAdHocTeam**
## Description of change

Add drawer for topic guidance.

## How to test

Create a goal and objective clicking the topic guidance link should open the drawer with the text from the jira.

Create an d

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1348

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [x] Update JIRA ticket status

### CODE REVIEW

**Comment by thewatermethod** [comment]
Screenshot 2023-03-24 at 10 55 22 AM
Screenshot 2023-03-24 at 10 55 09 AM

Headings are the same size as the text in the mockup. Also the (1) & (2) looks a little funky vs the mockup

File: frontend/src/components/GoalForm/TopicsInfoList.js
Comment (GarrettEHill) [inline frontend/src/components/GoalForm/TopicsInfoList.js:1]: Could we add the info to the DB and pull this data via an api as to not duplicate the list in multiple places requiring more maintenance when topic changes happen in the future

Comment (AdamAdHocTeam) [reply frontend/src/components/GoalForm/TopicsInfoList.js:1]: I think that would be more of a different ticket.

File: frontend/src/components/GoalForm/TopicsInfoList.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  },
+];
+
+export default function TopicsInfoList() {
+  return (
+    <>
+      {
+     topicInfo.map((t) => (
+       <>
+         <span className="smart-hub-topics-info-header margin-bottom-0">{t.title}</span>
```
Comment (thewatermethod) [inline frontend/src/components/GoalForm/TopicsInfoList.js:104]:
```suggestion
         <span className="smart-hub-topics-info-header margin-bottom-0 usa-prose text-bold">{t.title}</span>
```

File: frontend/src/components/GoalForm/TopicsInfoList.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+         <p className="smart-hub-topics-info-text margin-top-0" style={{ whiteSpace: 'break-spaces' }}>
```
Comment (thewatermethod) [inline frontend/src/components/GoalForm/TopicsInfoList.js:105]:
```suggestion
         <p className="smart-hub-topics-info-text margin-top-0 usa-prose" style={{ whiteSpace: 'break-spaces' }}>
```

File: frontend/src/components/GoalForm/TopicsInfoLIst.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.smart-hub-topics-info-header {
+  font-size: 16px;
```
Comment (thewatermethod) [inline frontend/src/components/GoalForm/TopicsInfoLIst.scss:2]: if you incorporate the two suggestions I made in your template you can delete this file

Comment (AdamAdHocTeam) [reply frontend/src/components/GoalForm/TopicsInfoLIst.scss:2]: Thanks!

Comment (thewatermethod) [reply frontend/src/components/GoalForm/TopicsInfoLIst.scss]: no - thank YOU

**Review by thewatermethod — APPROVED** [review]

**Pull request merged**